int64 nodeID
float64 velocity

